training/nc_train.xml

```diff
@@ -0,0 +1,55 @@
+  <AddressString><AddressNumber>2153</AddressNumber> <StreetName>U</StreetName> <StreetName>S</StreetName> <StreetName>401</StreetName> <StreetNamePostType>HWY</StreetNamePostType> <StreetNamePostDirectional>S</StreetNamePostDirectional> <PlaceName>LOUISBURG</PlaceName> <ZipCode>27549</ZipCode></AddressString>
+  <AddressString><AddressNumber>2471</AddressNumber> <StreetNamePreDirectional>S</StreetNamePreDirectional> <NotAddress>AND</NotAddress> <StreetNamePreDirectional>W</StreetNamePreDirectional> <StreetName>FARM</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>HICKORY</PlaceName> <ZipCode>28602</ZipCode></AddressString>
+  <AddressString><AddressNumber>2454</AddressNumber> <StreetNamePreDirectional>S</StreetNamePreDirectional> <NotAddress>AND</NotAddress> <StreetNamePreDirectional>W</StreetNamePreDirectional> <StreetName>FARM</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>HICKORY</PlaceName> <ZipCode>28602</ZipCode></AddressString>
+  <AddressString><AddressNumber>5638</AddressNumber> <StreetName>RIVER</StreetName> <StreetNamePostType>RD</StreetNamePostType> <StreetName>LIBERTY</StreetName> <StreetName>GROVE</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>NORTH</PlaceName> <PlaceName>WILKESBORO</PlaceName> <ZipCode>28659</ZipCode></AddressString>
+  <AddressString><AddressNumber>280</AddressNumber> <StreetName>C</StreetName> <StreetName>AND</StreetName> <StreetName>N</StreetName> <StreetName>SMITH</StreetName> <StreetName>MILL</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>STONEVILLE</PlaceName> <ZipCode>27048</ZipCode></AddressString>
+  <AddressString><AddressNumber>110</AddressNumber> <StreetName>ALEXANDER</StreetName> <StreetNamePostType>ST</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>3</OccupancyIdentifier> <PlaceName>ELIZABETHTOWN</PlaceName> <ZipCode>28337</ZipCode></AddressString>
+  <AddressString><AddressNumber>48</AddressNumber> <StreetName>ROBERT</StreetName> <StreetName>N</StreetName> <StreetName>BOWEN</StreetName> <StreetNamePostType>DR</StreetNamePostType> <PlaceName>WHITEVILLE</PlaceName> <ZipCode>28472</ZipCode></AddressString>
+  <AddressString><AddressNumber>4026</AddressNumber> <StreetName>MARK</StreetName> <StreetName>N</StreetName> <StreetName>SMITH</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>DEEP</PlaceName> <PlaceName>RUN</PlaceName> <ZipCode>28525</ZipCode></AddressString>
+  <AddressString><AddressNumber>1031</AddressNumber> <StreetName>21ST</StreetName> <StreetNamePostType>ST</StreetNamePostType> <StreetNamePostDirectional>NW</StreetNamePostDirectional> <StreetNamePostType>LN</StreetNamePostType> <PlaceName>HICKORY</PlaceName> <ZipCode>28601</ZipCode></AddressString>
```

Comment:
Great! Thanks for the speedy replies to my comments, @JakeStevens and @fgregg. Sounds like we're all good, so I'm going to go ahead and pull this in.